tests/js/spec/components/smartSearchBar/actions.spec.jsx

```diff
@@ -0,0 +1,104 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
```

Comment:
Could this new test file be in typescript?

Reply:
I didn't want to ignore stuff so I gave up on trying